deps/poulet4_Ccomp/Petr4Runtime.h

```diff
@@ -7,12 +7,13 @@
 
 enum p4int {FIXBIT, FIXINT};
 
-struct packet_in {
-  void *in;
-};
-struct packet_out{
-  void *out;
-};
+typedef struct packet_in {
+  int *in;
```

Comment:
should this be unsigned char and not int?

Reply:
yes you are right